### PR TITLE
Outline the migration process

### PR DESCRIPTION
**Pull request by teemow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+ * Migrate the CRs
+ * Once the CRs are ready hand them over to the CAPI controllers
+ * New nodes will be created
+ * Drain and remove the old master once the new one is there
+ * Edit the coredns deployment to fix the volume definition (not sure why it's broken)
+ * drain and remove old node pool
+
+### Cleanup Phase
+
+ * Remove the old CA fromt the etcd bundle
```
Comment (whites11) [inline README.md:22]:
```suggestion
 * Remove the old CA from the etcd bundle
```

**Review by whites11 — APPROVED** [review]
Seems a good starting point

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
-# capi-migration-controller
\ No newline at end of file
+# capi-migration-controller
```
Comment (kopiczko) [inline README.md:1]: Let's fix that :D

```suggestion
# capi-migration
```

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+ * Drain and remove the old master once the new one is there
+ * Edit the coredns deployment to fix the volume definition (not sure why it's broken)
+ * drain and remove old node pool
+
+### Cleanup Phase
+
+ * Remove the old CA from the etcd bundle
+ * Roll the masters again
+
+Errors still to be solved:
```
Comment (kopiczko) [inline README.md:25]:
```suggestion
### Errors still to be solved
```

**Review by kopiczko — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+
+ * externalDNS crashes
+worker nodes still have useManagedIdentity set to false despite the AzureMachineTemplate having it set to SystemAssigned  (this is likely the cause for external-dns crash listed above)
+ * PVC are not being provisioned
+ * Load balancer has issues
```
Comment (whites11) [inline README.md:30]:
```suggestion
 * Load balancer has issues (might be related to https://github.com/kubernetes-sigs/cloud-provider-azure/issues/363)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,30 @@
...
+
+### Cleanup Phase
+
+ * Remove the old CA from the etcd bundle
+ * Roll the masters again
+
+### Errors still to be solved
+
+ * externalDNS crashes
+worker nodes still have useManagedIdentity set to false despite the AzureMachineTemplate having it set to SystemAssigned  (this is likely the cause for external-dns crash listed above)
```
Comment (teemow) [inline README.md:28]:
```suggestion
 * worker nodes still have `useManagedIdentity` set to `false` despite the `AzureMachineTemplate` having it set to `SystemAssigned`  (this is likely the cause for external-dns crash listed above)
```

**Pull request merged**